tests/integration/controller/manifest/manifest_controller_test.go

```diff
@@ -155,6 +163,43 @@ var _ = Describe(
 	},
 )
 
+var _ = Describe(
+	"Rendering manifest install layer from tar", Ordered, func() {
+		ociTempDir := "main-dir"
+		installName := filepath.Join(ociTempDir, "installs")
+		setupTestEnvironment(ociTempDir, installName, v1beta2.MediaTypeDir)
+
+		Context("Given a Manifest CR", func() {
+			It("When Manifest CR contains a valid install OCI image specification",
+				func() {
+					manifest := testutils.NewTestManifest("oci")
+
+					Eventually(testutils.WithValidInstallImageSpecFromTar(ctx, kcpClient, installName,
+						manifestTarPath,
+						serverAddress, false, false), standardTimeout, standardInterval).
+						WithArguments(manifest).
+						Should(Succeed())
+					By("Then Manifest CR is in Ready State", func() {
+						Eventually(testutils.ExpectManifestStateIn(ctx, kcpClient, shared.StateReady),
+							standardTimeout, standardInterval).
+							WithArguments(manifest.GetName()).
+							Should(Succeed())
+					})
+					By("And OCI-Sync-Ref Annotation exists", func() {
```

Comment:
Please don't introduce this `"sync-oci-ref"` check, it's annotation will deprecated soon.